lib/data/parse-test-csv-row.js

```diff
@@ -216,12 +218,16 @@ function parseTestCSVRowV2({ tests, assertions, scripts, commands }) {
             at => at.key === key && at.settings === (command.settings || 'defaultMode')
           )
         ) {
+          const assertionExceptions = setDefaultAssertionExceptionsFromTest(
```

Comment:
`setDefaultAssertionPriority` seems like a more appropriate name here

Reply:
@howard-e addressed this in https://github.com/w3c/aria-at/pull/1053/commits/1373d76e0d2caa58c5b2f5d746e84655b90ae0d7